apps/api/src/auth/auth.service.ts

```diff
@@ -220,4 +221,52 @@ export class AuthService {
       where: { email: registerDto.email },
     })
   }
+
+  async updateUser(keycloakId: string, updateDto: UpdatePersonDto) {
+    await this.authenticateAdmin()
+    await this.admin.users.update(
+      { id: keycloakId },
+      {
+        username: updateDto.email,
+        email: updateDto.email,
+        firstName: updateDto.firstName,
+        lastName: updateDto.lastName,
+        emailVerified: true,
+        requiredActions: [],
+        enabled: true,
```

Comment:
I'm not sure if `updateUser` should set the `enable` flag to true. 

If we want to mark an account as disabled they will have the option to update their name and get enabled again by themselves.